Source/EasyNetQ/Consumer/InternalConsumer.cs

```diff
@@ -123,6 +125,13 @@ private void Cancel()
             if(consumerCancelled != null) consumerCancelled(this, new ConsumerEventArgs(ConsumerTag));
         }
 
+        private void FireConsumerLost()
```

Comment:
I've  used another naming convention. I will change it to RaiseConsumerLost, and subsequently define Durable as true by default in ResponderConfiguration, to avoid breaking change.